app/utils/answersHelpers/ConsignmentAnswersHelper.scala

```diff
@@ -360,11 +360,7 @@ class ConsignmentAnswersHelper(userAnswers: UserAnswers)(implicit messages: Mess
           val helper = new HouseConsignmentAnswersHelper(userAnswers, index)
           val rows = Seq(
             helper.consignorName,
-            helper.consignorIdentification,
-            helper.consigneeName,
-            helper.consigneeIdentification,
-            helper.consigneeCountry,
-            helper.consigneeAddress
+            helper.consignorIdentification
```

Comment:
Where has the consignment consignee gone?

Reply:
It has been moved into it's own h2 as a section

Reply:
Ahh gotcha. I see now this is the house consignment consignee. All good